DuckDuckGo/TabViewController.swift

```diff
@@ -108,6 +108,8 @@ class TabViewController: UIViewController {
     // In certain conditions we try to present a dax dialog when one is already showing, so check to ensure we don't
     var isShowingFullScreenDaxDialog = false
     
+    var temporaryDownload: Download?
```

Comment:
Thinking about it, maybe a more descriptive name would be better here. Like `currentlyPreviewedFileDownload` or `downloadForCurrentlyPreviewedFile` or `downloadForPreviewedFile` or `temporaryDownloadForPreviewedFile`

Reply:
I like `temporaryDownloadForPreviewedFile` thanks